example-scripts/voctolight/lib/plugins/rpi_gpio.py

```diff
@@ -0,0 +1,32 @@
+# Plugin to provide a tally light interface for a Raspberry Pi's GPIO
+DO_GPIO = True
+try:
+    import RPi.GPIO as GPIO
+    GPIO.setmode(GPIO.BOARD)
+except ImportError:
+    DO_GPIO = False
+
+__all__ = ['RpiGpio']
+
+class RpiGpio:
+    def __init__(self, config):
+        if not DO_GPIO:
+            raise ValueError('RpiGpio will not work on this platform. Is RPi.GPIO installed?')
+
+        # We don't need an explicit event loop pump
+        self.pump = lambda: None
+
+        all_gpios = [int(i) for i in config.get('rpi', 'gpios').split(',')]
+        self.gpio_port = int(config.get('rpi', 'gpio_red'))
```

Comment:
I see that this is not your code, but "red" is not an adequate name for this config property. It should better be called 'primary'. I would prefer If you would rename it and add a Fallback here, warning about the deprecation of `red` and the new name.

Reply:
I agree – but as you point out, this is existing code, and should be fixed in a separate PR rather than complicating things here.